sys/subprocess_unsafe_linux.go

```diff
@@ -0,0 +1,30 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package sys
+
+import (
+	_ "unsafe" // required for go:linkname.
+)
+
+//go:linkname beforeFork syscall.runtime_BeforeFork
```

Comment:
Is this officially supported across different versions of Go?

Reply:
```
# https://github.com/golang/go/blob/release-branch.go1.8/src/syscall/exec_linux.go#L46
func runtime_BeforeFork()
func runtime_AfterFork()
```

```
# https://github.com/golang/go/blob/release-branch.go1.9/src/syscall/exec_linux.go#L52
func runtime_BeforeFork()
func runtime_AfterFork()
func runtime_AfterForkInChild()
```
1.9 or higher supports the three features. But, I think go1.10 can work well than go1.9 since the `runtime.LockOSThread` has been improved. 

> more input from https://golang.org/doc/go1.10#runtime